webpack/assets/javascripts/react_app/redux/index.js

```diff
@@ -8,7 +8,7 @@ import { IntervalMiddleware, APIMiddleware } from './middlewares';
 
 let middleware = [thunk, IntervalMiddleware, APIMiddleware];
 
-const useLogger = () => {
+const logger = () => {
```

Comment:
Too bad the Hooks rule won't allow non-Hook functions whose names start with `use`.

How about `logReduxToConsole` or something more descriptive?